<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tagsDialog</class>
 <widget class="QDialog" name="tagsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Tags</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DeClutter.qrc">
    <normaloff>:/images/DeClutter.ico</normaloff>:/images/DeClutter.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="toolTip">
          <string>Add Tag</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="DeClutter.qrc">
           <normaloff>:/images/icons/tag.svg</normaloff>:/images/icons/tag.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addGroupButton">
         <property name="toolTip">
          <string>Add Group</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="DeClutter.qrc">
           <normaloff>:/images/icons/folder.svg</normaloff>:/images/icons/folder.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="toolTip">
          <string>Delete</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="DeClutter.qrc">
           <normaloff>:/images/icons/trash.svg</normaloff>:/images/icons/trash.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorButton">
         <property name="toolTip">
          <string>Select Color</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="DeClutter.qrc">
           <normaloff>:/images/icons/brush.svg</normaloff>:/images/icons/brush.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="TreeView" name="treeView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>tag_groups</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DeClutter.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tagsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tagsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
